src/main/java/com/metamx/emitter/core/ParametrizedUriEmitter.java

```diff
@@ -115,7 +121,9 @@ public void emit(Event event)
   @LifecycleStop
   public void close() throws IOException
   {
-    innerLifecycle.stop();
+    if (!stopFlag.getAndSet(true)) {
```

Comment:
Actually, there's a race condition here. If two things call stop, the second one can return before the first finishes, giving the false impression that `stop` on the ilfecycle completed